### ISSUE TITLE
lab-04-4-tf_reader_linear_regression.py

### ISSUE THREAD

**Issue by qoocrab** [hunkim/DeepLearningZeroToAll#230]
'lab-04-4-tf_reader_linear_regression.py' code needs to be improved. 
The 'Queue-based input pipelines' used in the code is going to be removed from Tensorflow. 
So there is a warning message suggesting to change to 'tf.data' format.

WARNING:tensorflow:From D:/study-deeplearning/lab_04_04_TF_Reader_Linear_Regression.py:9: string_input_producer (from tensorflow.python.training.input) is deprecated and will be removed in a future version.
Instructions for updating:
Queue-based input pipelines have been replaced by `tf.data`. Use `tf.data.Dataset.from_tensor_slices(string_tensor).shuffle(tf.shape(input_tensor, out_type=tf.int64)[0]).repeat(num_epochs)`. If `shuffle=False`, omit the `.shuffle(...)`.

**Comment by tcratius** [comment]
https://stackoverflow.com/questions/57381430/synonym-of-type-is-deprecated-in-a-future-version-of-numpy-it-will-be-underst

See Stakes Overflow, and downgrade numpy 1.14.5
